Automattic/woocommerce-payments
# fix: PMs listed in disable confirmation modal

Fixes #

#### Changes proposed in this Pull Request









When the "disable confirmation modal" is displayed, we state: "Payment methods that need WooPayments", and list a bunch of payment methods.
However, the list of payment methods displayed is not representative of the list of payment methods that are enabled in the merchant's settings.

Updating, so that the list of payment methods is accurately showing the list of enabled payment methods (including Link, WooPay, and GooplePay)


Before:
![2024-02-20 10 00 09](https://github.com/Automattic/woocommerce-payments/assets/273592/b8ef5e65-4a1b-480e-ac50-e7372941136d)

After:
![2024-02-20 09 59 31](https://github.com/Automattic/woocommerce-payments/assets/273592/b7fb1e7f-355e-46fb-98d3-2787e7e558b5)


#### Testing instructions





- Go to the WC settings page at http://wcpay.test/wp-admin/admin.php?page=wc-settings&tab=checkout
- Attempt to disable WooPayments
- The list of payment methods displayed in the confirmation modal should be accurately representing the list of payment methods that are enabled

-------------------


**Post merge**
